bigquery_etl/cli/query.py

```diff
@@ -31,80 +39,9 @@
 from .dryrun import dryrun
 
 QUERY_NAME_RE = re.compile(r"(?P<dataset>[a-zA-z0-9_]+)\.(?P<name>[a-zA-z0-9_]+)")
-SQL_FILE_RE = re.compile(
-    r"^.*/([a-zA-Z0-9-]+)/([a-zA-Z0-9_]+)/([a-zA-Z0-9_]+_v[0-9]+)/"
-    r"(?:query\.sql|part1\.sql|script\.sql)$"
-)
 VERSION_RE = re.compile(r"_v[0-9]+")
 
 
-def _queries_matching_name_pattern(pattern, sql_path, project_id):
-    """Return paths to queries matching the name pattern."""
-    sql_path = Path(sql_path)
-    if project_id is not None:
-        sql_path = sql_path / project_id
-
-    all_sql_files = Path(sql_path).rglob("*.sql")
-    sql_files = []
-
-    for sql_file in all_sql_files:
-        match = SQL_FILE_RE.match(str(sql_file))
-        if match:
-            project = match.group(1)
-            dataset = match.group(2)
-            table = match.group(3)
-            query_name = f"{project}.{dataset}.{table}"
-            if fnmatchcase(query_name, f"*{pattern}"):
-                sql_files.append(sql_file)
-            elif project_id and fnmatchcase(query_name, f"{project_id}.{pattern}"):
-                sql_files.append(sql_file)
-
-    return sql_files
-
-
-sql_dir_option = click.option(
-    "--sql_dir",
-    help="Path to directory which contains queries.",
-    type=click.Path(file_okay=False),
-    default="sql",
-    callback=is_valid_dir,
-)
-
-use_cloud_function_option = click.option(
-    "--use_cloud_function",
-    "--use-cloud-function",
-    help=(
-        "Use the Cloud Function for dry running SQL, if set to `True`. "
-        "The Cloud Function can only access tables in shared-prod. "
-        "If set to `False`, use active GCP credentials for the dry run."
-    ),
-    type=bool,
-    default=True,
-)
-
-
-def respect_dryrun_skip_option(default=True):
-    """Generate a respect_dryrun_skip option."""
-    flags = {True: "--respect-dryrun-skip", False: "--ignore-dryrun-skip"}
-    return click.option(
-        f"{flags[True]}/{flags[False]}",
-        help="Respect or ignore dry run SKIP configuration. "
-        f"Default is {flags[default]}.",
-        default=default,
-    )
-
-
-def project_id_option(default=None):
-    """Generate a project-id option, with optional default."""
-    return click.option(
-        "--project-id",
-        "--project_id",
-        help="GCP project ID",
-        default=default,
-        callback=is_valid_project,
-    )
-
-
```

Comment:
Move to `cli.common` since these CLI parameters are also used for other commands.